lib/hacluster.pm

```diff
@@ -684,7 +689,7 @@ sub ha_export_logs {
     $crm_log_name = $1;
     upload_logs("$crm_log_name", failok => 1);
 
-    script_run "crm configure show > /tmp/crm.txt";
+    record_info('crm configure show', 'Failed to run "crm configure show"', result => 'fail') if (script_run("crm configure show > /tmp/crm.txt") != 0);
```

Comment:
maybe use `tee` in place of `>`. Doing so the output will be visible in th eopenQA serial_terminal log

Reply:
I didn't change the command `script_run "crm configure show > /tmp/crm.txt";` only checking the exit code.  This output will be uploaded as a log file `crm.txt`